config/env_spec_request.go

```diff
@@ -54,6 +57,7 @@ const (
 	PathNamespace              = "path"
 	HeaderNamespace            = "headers"
 	RequestPath                = "path"
+	OriginalPath               = "original_path"
```

Comment:
Hmm... "path" and "original_path" seems like it will be confusing because "path" is also the original path without the base path, and there will be a further target path after it's processed. Rethinking the naming here would be worthwhile, I think.